timeseries/src/autogluon/timeseries/models/ensemble/array_based/regressor/tabular.py

```diff
@@ -0,0 +1,127 @@
+import logging
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+from typing_extensions import Self
+
+from autogluon.tabular import TabularPredictor
+
+from .abstract import EnsembleRegressor
+
+logger = logging.getLogger(__name__)
+
+
+class TabularEnsembleRegressor(EnsembleRegressor):
+    """TabularPredictor ensemble regressor using AutoGluon-Tabular as a single
+    quantile regressor for the target.
+    """
+
+    def __init__(
+        self,
+        quantile_levels: list[float],
+        tabular_hyperparameters: Optional[dict] = None,
```

Comment:
This can be made more generic by having `tabular_init_kwargs` and `tabular_fit_kwargs`.

Reply:
I purposely hide these from the user. Even exposing hyperparameters is debatable imo. We only really had a good experience with GBM iirc and I would avoid exposing too many knobs that would break. For example, we don't want to expose the ability to set `problem_type="classification"` or turn bagging up to eleven, etc.